### PR TITLE
feat(feeds): add option to disable inlining of post content in feed.sh emails

### PR DESCRIPTION
**Pull request by jchenry**
Per Issue #7

### CODE REVIEW

**Comment by neurosnap** [comment]
Thanks for submitting this PR! I'm okay with the change. 

We probably want to disable showing the description as well in the template.

**Comment by jchenry** [comment]
@neurosnap updated to include disabling the description

File: shared/api.go [hunk 1]
```diff
@@ -41,6 +41,10 @@ func GetUsernameFromRequest(r *http.Request) string {
 	return subdomain
 }
 
+func ServeRobots() http.HandlerFunc {
```
Comment (neurosnap) [inline shared/api.go:44]: Is this being used anywhere?

File: shared/listparser.go [hunk 2]
```diff
@@ -51,6 +51,7 @@ type ListMetaData struct {
 	ListType       string // https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type
 	DigestInterval string
 	Email          string
+	InlineContent  string // allows content inlining to be disabled in feeds.sh emails
```
Comment (neurosnap) [inline shared/listparser.go:54]: This has to be hooked up to [TokenToMetaField](https://github.com/picosh/pico/blob/0d7a17435d3cf31e28ee9b1bb796a581d0582ae9/shared/listparser.go#L100) and you can make it a `bool`, set the default to true, and perform proper type checks on it.

**Review by neurosnap — CHANGES_REQUESTED** [review]
Close!

File: shared/api.go (hunk 1 above)
Comment (jchenry) [reply shared/api.go:44]: How did that get there? Was a commit from another branch that somehow leaked in. will remove.

**Review by neurosnap — APPROVED** [review]

**Pull request merged**